asserts/gpgkeypairmgr.go

```diff
@@ -183,3 +223,83 @@ func (gkm *GPGKeypairManager) sign(fingerprint string, content []byte) ([]byte,
 	}
 	return out, nil
 }
+
+type gpgKeypairInfo struct {
+	pubKey      PublicKey
+	fingerprint string
+}
+
+func (gkm *GPGKeypairManager) findByName(name string) (*gpgKeypairInfo, error) {
+	stop := errors.New("stop marker")
+	var hit gpgKeypairInfo
```

Comment:
That seems kind of inconsistent with the existing GPGKeypairManager and indeed all the various KeypairManager interfaces.